lib/shared/config/convert-config.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var path = require('path');
+var logger = require('fancy-log');
+var chalk = require('chalk');
+
+module.exports = {
+
+  description: function(value, configFile) {
```

Comment:
Can that be done inline instead of an abstraction like this (similar to configBase)?